xc/device.ml

```diff
@@ -1725,22 +1725,6 @@ let start_vnconly (task: Xenops_task.t) ~xs ~dmpath ?timeout info domid =
 
 (* suspend/resume is a done by sending signals to qemu *)
 let suspend (task: Xenops_task.t) ~xs ~qemu_domid domid =
-	let suspend_vgpu () = match Vgpu.pid ~xs domid with
-	| None -> debug "vgpu: no process running"
-	| Some vgpu_pid -> begin
-		let tag = Printf.sprintf "(domid = %d pid = %d)" domid vgpu_pid in
-		debug "vgpu: suspending vgpu with SIGHUP %s" tag;
-		try
-			Unixext.kill_and_wait
-				~signal:Sys.sighup ~timeout:(!Xenopsd.vgpu_suspend_timeout) vgpu_pid;
```

Comment:
After this change, `vgpu_suspend_timeout` is no longer used by anything in xenopsd. If it is no longer needed for future changes, you could also remove that.